api/src/main/resources/liquibase-update-to-latest.xml

```diff
@@ -796,6 +796,189 @@
 			<column name="uuid" type="char(38)" />
 		</addColumn>
 	</changeSet>
+	<changeSet author="vshankar" id="201609171146-1">
+		<preConditions onFail="MARK_RAN">
+            <not><columnExists tableName="cohort_member" columnName="cohort_member_id"/></not>
```

Comment:
The correct precondition should instead check if the foreign key exists i.e `<foreignKeyConstraintExists foreignKeyName="member_patient" />`